prod/westeurope/cgn/functions_cgn/function_app/terragrunt.hcl

```diff
@@ -129,6 +129,9 @@ inputs = {
     REDIS_PASSWORD = dependency.redis.outputs.primary_access_key
 
     OTP_TTL_IN_SECONDS = 600
+
+    CGN_UPPER_BOUND_AGE  = 61
```

Comment:
So basically at 61 are you still considered young ? 
Maybe you are super young :-)